### ISSUE TITLE
Error when renaming file and trying to reload buffer

### ISSUE THREAD

**Issue by JosephFerano** [scrooloose/nerdtree#307]
Hi, I'm getting this error when I try to rename a file by using "(m)ove the current node" in the menu. It renames it correctly but throws an error when trying to reload the buffer.

````
Error detected while processing function nerdtree#invokeKeyMap..79..78..<SNR>38_showMenu..53..70..NERDTreeMoveNode..<SNR>49_promptToRenameBuffer:
line    4:
E172: Only one file name allowed: badd /Users/josephferano/Documents/Work/poptok/Assets/GUI Screens/User Login/UserLoginScreen.cs
Error detected while processing function nerdtree#invokeKeyMap..79..78..<SNR>38_showMenu..53..70:
line    6:
E171: Missing :endif
Error detected while processing function nerdtree#invokeKeyMap..79..78..<SNR>38_showMenu..53:
line   19:
E171: Missing :endif
Error detected while processing function nerdtree#invokeKeyMap..79..78:
line    3:
E171: Missing :endif
Error detected while processing function nerdtree#invokeKeyMap..79:
line   23:
E171: Missing :endif 
````

**Comment by chronzerg** [comment]
Bump. Here is my dump:

Error detected while processing function nerdtree#ui_glue#invokeKeyMap[1]..73[33]..72[3]..<SNR>26_showMenu[2]..47[19]..64[6]..NERDTreeMoveNode[22]..<SNR>28_promptToRenameBuffer:
line    3:
E121: Undefined variable: a:newFilename
E116: Invalid arguments for function fnameescape(a:newFilename)
E15: Invalid expression: fnameescape(a:newFilename)
Error detected while processing function nerdtree#ui_glue#invokeKeyMap[1]..73[33]..72[3]..<SNR>26_showMenu[2]..47[19]..64:
line    6:
E171: Missing :endif
Error detected while processing function nerdtree#ui_glue#invokeKeyMap[1]..73[33]..72[3]..<SNR>26_showMenu[2]..47:
line   19:
E171: Missing :endif
Error detected while processing function nerdtree#ui_glue#invokeKeyMap[1]..73[33]..72:
line    3:
E171: Missing :endif
Error detected while processing function nerdtree#ui_glue#invokeKeyMap[1]..73:
line   33:
E171: Missing :endif

**Comment by ladislas** [comment]
yep, still have the same error when moving a file.

**Comment by PhilRunninger** [comment]
This should be fixed now, with pull request #623, which addresses a duplicate issue #622.
